src/scripts/lib/get-geometry-column-name.ts

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Returns the name of the geometry column
+ */
+export default function(data: string[][]): string {
```

Comment:
I thought we would name these functions, e.g getGeometryColumn
and not use unnamed exports.

Reply:
@russellquong  I wanted to be consistent among the code base first, but I could certainly make another PR changing all default exports to namend exports